spec.bs

```diff
@@ -6340,9 +6362,11 @@ component, and top-level auctions may specify which signals to load by the `adSl
 HTTP response header.</h3>
 
 The \`<dfn http-header><code>Ad-Auction-Additional-Bid</code></dfn>\` response header provides value
-of a string in the format of `<auction nonce>:<base64-encoding of the signed additional bid>`, which
-corresponds to a single [=additional bid=]. The response may include more than one [=additional bid=]
-by specifying multiple instances of the [:Ad-Auction-Additional-Bid:] response header.
+of a string in the format of `<auction nonce>:<seller nonce>:<base64-encoding of the signed

```

Comment:
Related to the comment at the end; though we've defined 'auction nonce' before this, we haven't defined 'seller nonce' before this. Can we add a sentence that says, "In the format that provides a seller nonce, the seller nonce should be the canonical string representation of a [=version 4 UUID=]", or "[=version 4 UUID=] [=string=]" based on the answer to the question above.

Reply:
Done, although I just link to the seller nonce definition to avoid duplicating the language.